modules/twinkleconfig.js

```diff
@@ -200,6 +200,24 @@ Twinkle.config.sections = [
 				name: 'dialogLargeFont',
 				label: 'Use larger text in Twinkle dialogs',
 				type: 'boolean'
+			},
+
+			// Twinkle.config.disabledModules (array)
+			{
+				name: 'disabledModules',
+				label: 'Turn off the selected Twinkle modules',
+				helptip: 'Anything you select here will NOT be loaded by Twinkle, so act with care. Uncheck to reenable.',
```

Comment:
```suggestion
				helptip: 'Anything you select here will NOT be available for use, so act with care. Uncheck to re-enable.',
```
Or something.

The original isn't technically true as everything is still loaded. It's just that the code isn't executed. Users shouldn't get the impression that disabling TW modules will result in faster page loads (or lesser bandwidth consumption).

[nitpick] `reenable` → `re-enable` maybe?

Reply:
I think now is actually the time to commit Twinkle to the New Yorker standard: `reënable`

Reply:
The nuance is probably lost on most folks, and truth be told "available for use" isn't quite right either, but it's closer and I take the point.

How's reactivate?

Reply:
> I think now is actually the time to commit Twinkle to the New Yorker standard: `reënable`

You're [kidding me](https://www.google.com/search?q=re%C3%ABnable+site%3Anewyorker.com)?

Reply:
[Diaresis all the things!](https://www.newyorker.com/culture/culture-desk/the-curse-of-the-diaeresis) 😀 😆 😀